waspc/src/Wasp/AI/GenerateNewProject/Page.hs

```diff
@@ -107,11 +107,18 @@ generatePage newProjectDetails entityPlans queries actions pPlan = do
 
         Example of such JSON:
         {
-          "pageWaspDecl": "route ExampleRoute { path: \"/\", to: ExamplePage }\npage ExamplePage {\n  component: import { ExamplePage } from \"@client/ExamplePage.jsx\",\n  authRequired: true\n}",
+          "pageWaspDecl": "route ExampleRoute { path: \"/\", to: ExamplePage }\npage ExamplePage {\n  component: import ExamplePage from \"@client/ExamplePage.jsx\",\n  authRequired: true\n}",
           "pageJsImpl": "JS imports + React component implementing the page.",
         }
         There should be no other text in the response.
 
+        When writing the javascript implementation, make sure to always use the default export. 
+        Concretely, define the main component with `const ${pageName} = () => {...}`,
+        and then at the end export it with `export default ${pageName}`.
+        It is also really important that the ${pageName} is then imported as a default import. 
+        Concretely, in "pageWaspDecl", use `import ${pageName}` instead of `import { ${pageName} }`.
```

Comment:
```suggestion
        It is also really important that the ${pageName} is then imported as a default import in "pageWaspDecl": use `import ${pageName}` instead of `import { ${pageName} }`.
```

Reply:
My understanding is that LLMs are extremely insensitive when it comes to small syntactic things (such as here -- using `:` instead of `. Concretely, `. Why "extremely"? See for example the following paper: https://arxiv.org/abs/2311.18805

Having said that, there's no good reason to keep it confusing for humans. I'll implement the suggested change.

Reply:
Makes sense, I would also expect GPT to not care much, but while at it why not make it as clear as possible :), at least for humans as you said!